src/App.tsx

```diff
@@ -10,33 +11,17 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [select, setSelect] = useState(
+    { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
+  );
```

Comment:
and maybe we can rewrite it and save only selectedId (not the whole object). What do you think?